metrics/report/report_dockerfile/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # We would have used the 'verse' base, that already has some of the docs processing
 # installed, but I could not figure out how to add in the extra bits we needed to
 # the lite tex version is uses.
-FROM rocker/tidyverse
+FROM rocker/tidyverse:3.6.0
 
 # Version of the Dockerfile
 LABEL DOCKERFILE_VERSION="1.1"
```

Comment:
Does this version number need bumping too @grahamwhaley?

Reply:
yes. the intention is to ensure that if somebody has an already built image on their machine, they will get the update pulled in and built (otherwise docker will probably use the already built/cached image).
Nice catch @jodh-intel